src/org/zaproxy/zap/extension/codedx/resources/Messages.properties

```diff
@@ -22,10 +22,13 @@ codedx.error.http.415       = The server returned Error 415: Unsupported Media T
 codedx.error.http.other     = The response code is:
 codedx.error.client.failed  = An unknown error occurred while creating the HTTP client.\nMake sure the Server URL and API Key are correct, and that the server is online.
 codedx.error.client.invalid = The Server URL is not a valid URL. Please check that it is correct.
+codedx.error.httpMessage    = No http message found for alert id = 
```

Comment:
If you use `No http message found for alert id = {0}` then it's usage can be: `logger.error(Constant.messages.getString("codedx.error.httpMessage"), alert.getAlertId()));`

I guess HTTP should be fullcaps in the message as well.

Also is `error` the correct level? Does it prevent execution? Or is it simply sub-optimal? (in which case `warn` is probably fine)